src/System.Console/src/System.Console.csproj

```diff
@@ -51,12 +41,13 @@
       <Link>Common\System\IO\EncodingHelper.cs</Link>
     </Compile>
   </ItemGroup>
-  <!-- Windows : UAP -->
-  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(TargetGroup)' == 'netcore50' ">
+  <!-- Windows : UAP 
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' And '$(TargetGroup)' == 'netcore50' ">
```

Comment:
Nit: It would be better to change the condition to uap101 instead of commenting it out